Library/Tracking/Koala.swift

```diff
@@ -61,6 +63,7 @@ public final class Koala {
     case forgotPassword = "forgot_password_screen" // ResetPasswordViewController
     case login = "login_screen" // LoginViewController
     case loginTout = "login_or_signup_screen" // LoginToutViewController
+    case managePledgeScreen = "manage_pledge_screen"
```

Comment:
Please add the `// ManagePledgeViewController` comment next to this just for consistency 👍